### PR TITLE
Replace Keystone Admin UI popover button in production with Sign-out button

### PR DESCRIPTION
**Pull request by Achisingh**
The keystone Admin UI popover button that consisted of Keystone links to API Explorer, GitHub Repository and Keystone Documentation, present at the top of the left side-bar is replaced with only the `Sign out` button in `production mode` after authentication.
Screenshot 2022-05-23 at 12 54 37 pm

### CODE REVIEW

**Comment by Achisingh** [comment]
Hi @raveling 
This is what the UI looks after replacing the popover button with Sign-out in production. As you will see the text is coming very close to the button. Any suggestions how to go about this?
Screenshot 2022-05-19 at 2 59 49 pm

For reference, this is how the text and the popover button are in dev mode.
Screenshot 2022-05-20 at 11 27 59 am

File: packages/core/src/admin-ui/components/Navigation.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const AuthenticatedItemDialog = ({ item }: { item: AuthenticatedItem | undefined
     >
       {item && item.state === 'authenticated' ? (
         <div css={{ fontSize: typography.fontSize.small }}>
-          Signed in as <strong>{item.label}</strong>
+          Signed in as <strong css={{ display: 'block' }}>{item.label}</strong>
```
Comment (Achisingh) [inline packages/core/src/admin-ui/components/Navigation.tsx:79]: This is how the `Signed in as` looks like
Screenshot 2022-05-23 at 12 46 50 pm

**Comment by Achisingh** [comment]
This is how the `Signed in as` look like after stylus update
Screenshot 2022-05-23 at 12 46 50 pm

**Comment by raveling** [comment]
That's acceptable from a Design point of view. Thanks @Achisingh !

**Review by mitchellhamilton — APPROVED** [review]

**Comment by mitchellhamilton** [comment]
Could you update the test at https://github.com/keystonejs/keystone/blob/fd8d4a64f280145a3963f70f1d0cd23930e7109e/tests/examples-smoke-tests/basic.test.ts#L14-L20? The test will need to do different behavior depending on whether it's running the prod vs dev mode. You'll probably want to pass the `mode` variable to https://github.com/keystonejs/keystone/blob/fd8d4a64f280145a3963f70f1d0cd23930e7109e/tests/examples-smoke-tests/utils.ts#L154

**Comment by Achisingh** [comment]
> Could you update the test at
> 
> https://github.com/keystonejs/keystone/blob/fd8d4a64f280145a3963f70f1d0cd23930e7109e/tests/examples-smoke-tests/basic.test.ts#L14-L20
> 
> ? The test will need to do different behavior depending on whether it's running the prod vs dev mode. You'll probably want to pass the `mode` variable to
> https://github.com/keystonejs/keystone/blob/fd8d4a64f280145a3963f70f1d0cd23930e7109e/tests/examples-smoke-tests/utils.ts#L154

Done. Refer to: 
[https://github.com/keystonejs/keystone/pull/7546/commits/4c7b4713526280a6c1ded38c3308813300cb1442](url)

**Review by mitchellhamilton — APPROVED** [review]

**Pull request merged**